test-clients/src/main/java/com/hedera/services/bdd/suites/token/TokenTransactSpecs.java

```diff
@@ -482,6 +486,8 @@ public HapiApiSpec uniqueTokenTxnAccountBalance() {
 								.hasTokenBalance(A_TOKEN, 0),
 						getAccountBalance(FIRST_USER)
 								.hasTokenBalance(A_TOKEN, 1),
+						getTokenInfo(A_TOKEN)
```

Comment:
Why do you have check for the treasury if you only created the token and haven't updated the treasury? Isn't it unecessary